project/BuildSettings.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014-2020 Snowplow Analytics Ltd. All rights reserved.
+ * Copyright (c) 2014-2021 Snowplow Analytics Ltd. All rights reserved.
```

Comment:
I've always wondered about the use of these `Extend copyright notice` commits, for me we can update the end date at the same time as we update a file

Reply:
If I remember to do it when I edit a file - I do update it. But usually I realise most of the copyrights are outdated at the end of the release process.